### PR TITLE
Bug 1228929 - [TV] Seperate the Send-to-TV offline content from the iframe

### PR DESCRIPTION
**Pull request by Fischer-L**
This patch:
Separate the Send-to-TV offline content from the iframe to let smart-button work as expected because the document.registerElement method is unavailable inside iframe in the simulator.

p.s The bug 1228927 is to track this issue for the online content.

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=437582c358f57e098ce0bd25503200b7486687c4

File: tv_apps/fling-tutorial/js/index.js [hunk 1]
```diff
@@ -1,33 +1,48 @@
...
-  mozIframe.addEventListener('mozbrowserclose', () => {
-    window.close();
-  });
+    mozIframe.src = 'https://casttv.services.mozilla.com';
+    mozIframe.classList.remove('hidden');
+  } else {
+
+    var tutorial = new FTEWizard('flingTutorial');
+    tutorial.init({
+      container: document.body,
```
Comment (rexboy7) [inline tv_apps/fling-tutorial/js/index.js]: Use `offline-content` instead.

File: tv_apps/fling-tutorial/style/index.css [hunk 2]
```diff
@@ -16,7 +16,66 @@ html, body {
   background-color: black;
 }
 
-#mozIframe {
+body {
+  opacity: 1;
```
Comment (rexboy7) [inline tv_apps/fling-tutorial/style/index.css]: Move corresponding styles to `offline-content`.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=bd5d06143eff25ae84fc586b4e74a8c472d4de6b

**Pull request merged**